server/api/services/repo.service.js

```diff
@@ -8,8 +8,8 @@ const createRepo = async ({ user, name }) => {
   let result = 'Repo was created';
   const pathToRepo = path.resolve(`${gitPath}/${user}/${name}`);
   await NodeGit.Repository.init(pathToRepo.replace(/\\/g, '/'), 1)
-    .then(repo => {})
-    .catch(reasonForFailure => {
+    .then(() => {})
```

Comment:
```suggestion
    .then(() => {})
```
`.then()` тут не нужен, `.catch()` будет работать без него

Reply:
FIxed, thank you!